packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -1128,6 +1143,59 @@ class GoTrueClient {
     _currentUser = null;
   }
 
+  void mayStartBroadcastChannel() {
+    if (const bool.fromEnvironment('dart.library.html') && _broadcastSession) {
```

Comment:
It may be reasonable to add a try/catch block in this function. It can throw some unexpected errors, like `Uri.parse` and `messageRaw['event]`. What do you think?